centreon/src/Core/Dashboard/Infrastructure/Serializer/DashboardContactGroupsResponseNormalizer.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * Copyright 2005 - 2025 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Dashboard\Infrastructure\Serializer;
+
+use Core\Dashboard\Application\UseCase\FindDashboardContactGroups\Response\ContactGroupsResponseDto;
+use Core\Dashboard\Infrastructure\Model\DashboardGlobalRoleConverter;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+final class DashboardContactGroupsResponseNormalizer implements NormalizerInterface
+{
+    public function __construct(
+        private readonly ObjectNormalizer $normalizer,
+    ) {
+    }
+
+    public function supportsNormalization(mixed $data, ?string $format = null): bool
+    {
+        return $data instanceof ContactGroupsResponseDto;
```

Comment:
```suggestion
        return $data instanceof ContactGroupsResponseDto || $data instanceof ContactsResponseDto;
```
So you can remove your other normalizer as it does exactly the same normalization process.

We discuss about it with @a-launois but we should consider to regroup normalizer when many normalizer do exactly the same things and we know this behaviour will not change. we can discuss it further in backend sync

You can resolve if you won't apply

Reply:
Noted I'll apply this on futur PR